x-pack/plugins/file_upload/public/importer/geo/geojson_importer/geojson_importer.test.js

```diff
@@ -5,8 +5,23 @@
  * 2.0.
  */
 
+import {
+  ReadableStreamPolyfill,
+  BlobPolyfill,
+  FileReaderPolyfill,
+  FilePolyfill,
+} from '@loaders.gl/polyfills';
 import { GeoJsonImporter } from './geojson_importer';
 
+(function () {
+  // can not use installFilePolyfills because global.Blob exists, it just does not contain Blob.Blob.prototype.arrayBuffer
+  // installFilePolyfills only installs poly fill if global.Blob does not exist
```

Comment:
Can you add a small comment about removing this when we upgrade to Node 18 please?